lib/capybara/node/actions.rb

```diff
@@ -6,36 +6,32 @@ module Actions
       # @!macro waiting_behavior
       #   If the driver is capable of executing JavaScript, this method will wait for a set amount of time
       #   and continuously retry finding the element until either the element is found or the time
-      #   expires. The length of time +find+ will wait is controlled through {Capybara.default_max_wait_time}
+      #   expires. The length of time {Capybara::Node::Finders#find} will wait is controlled through {Capybara.configure default_max_wait_time}.
       #
-      #   @option options [false, true, Numeric] wait (Capybara.default_max_wait_time) Maximum time to wait for matching element to appear.
+      #   For each option details, see {Capybara::Node::Finders#find} options.
```

Comment:
I don't want this change - the `wait` parameter declaration should stay for each method this macro is included in

Reply:
Because this macro is almost same as the macro defined in `Finders` class, I want to be DRY. For it, I think that just adding the link to `Finders#find` options is good.
Generally, to maintain duplicate documents is hard. But this change is out of this PR's original purpose… 😓 

https://github.com/teamcapybara/capybara/blob/bcbefe7acceef14ca7b90c4aacfa0b9ce8520e20/lib/capybara/node/finders.rb#L11-L16